ts/main.ts

```diff
@@ -114,23 +112,20 @@ document.addEventListener('DOMContentLoaded', () => {
   }
 });
 
-const $p = document.querySelector('p');
+const $noEntries = document.querySelector('.no-entries');
 
 function toggleNoEntries(): void {
-  if (!$p) throw new Error('The $p query failed');
-
-  if ($p.classList.contains('no-entries') && data.entries.length > 0) {
-    $p.classList.add('hidden');
-  } else if ($p.classList.contains('no-entries') && data.entries === null) {
-    $p.classList.remove('hidden');
+  if (!$noEntries) throw new Error('the $noEntries query failed');
+  if ($noEntries.classList.contains('no-entries') && data.entries.length > 0) {
+    $noEntries.classList.add('hidden');
+  } else if (
+    $noEntries.classList.contains('no-entries') &&
+    data.entries === null
```

Comment:
I don't think `data.entries` will ever be null. It will either be an empty array or an array with values